autoload/ale/completion.vim

```diff
@@ -581,9 +581,14 @@ function! ale#completion#ParseLSPCompletions(response) abort
             continue
         endif
 
-        if get(l:item, 'insertTextFormat', s:LSP_INSERT_TEXT_FORMAT_PLAIN) is s:LSP_INSERT_TEXT_FORMAT_PLAIN
+        let l:insertTextFormat = get(l:item, 'insertTextFormat', s:LSP_INSERT_TEXT_FORMAT_PLAIN)
+
+        if l:insertTextFormat is s:LSP_INSERT_TEXT_FORMAT_PLAIN
         \&& type(get(l:item, 'textEdit')) is v:t_dict
             let l:text = l:item.textEdit.newText
+        elseif l:insertTextFormat is s:LSP_INSERT_TEXT_FORMAT_SNIPPET
```

Comment:
This is there we have fake snipped support. Still it is better than nothing as `label` can be used for auto-complete.

Reply:
It is safe to do this as below we have call to `ale#completion#Filter`.

Reply:
Not well versed in language servers so will have to ask @w0rp to take a look here.

Reply:
OK. Let's see what @w0rp has to say. It solves problem with this LSP but I'm pretty sure other LSPs will benefit accidentally without causing any accidental harm. Proper snippet support would be even better potentially, but that's really big task IMHO.

Reply:
Removed from this PR. Let's do this as separate PR.